src/core.js

```diff
@@ -427,7 +427,7 @@ imgix.setElementImage = function (el, imgUrl) {
  * @returns {string} url of an empty image
  */
 imgix.getEmptyImage = function () {
-  return imgix.versionifyUrl('https://assets.imgix.net/pixel.gif');
+  return 'https://assets.imgix.net/pixel.gif?ixjsv=' + imgix.version;
```

Comment:
In the interest of keeping the code DRY, we should keep this as `imgix.versionifyUrl`.